<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Jagseer-PC" name="com.prac.TestNG.PracTestNG" tests="10" failures="0" timestamp="17 Aug 2017 04:17:03 GMT" time="0.122" errors="1">
  <testcase name="dataP" time="0.015" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="display" time="0.005" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="prm" time="0.002" classname="com.prac.TestNG.PracTestNG">
    <error type="org.testng.TestNGException" message="
Parameter &#039;Name&#039; is required by @Test on method prm but has not been marked @Optional or defined
in C:\Users\Jagseer\AppData\Local\Temp\testng-eclipse--804795365\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'Name' is required by @Test on method prm but has not been marked @Optional or defined
in C:\Users\Jagseer\AppData\Local\Temp\testng-eclipse--804795365\testng-customsuite.xml
	at org.testng.internal.Parameters.createParameters(Parameters.java:199)
	at org.testng.internal.Parameters.createParameters(Parameters.java:443)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:569)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- prm -->
  <testcase name="dataP" time="0.013" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="dataP" time="0.005" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="dataP" time="0.051" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="funCheck" time="0.009" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="fn" time="0.010" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="dataP" time="0.009" classname="com.prac.TestNG.PracTestNG"/>
  <testcase name="dataP" time="0.003" classname="com.prac.TestNG.PracTestNG"/>
</testsuite> <!-- com.prac.TestNG.PracTestNG -->
